src/serial.c

```diff
@@ -228,6 +239,9 @@ void usart3_isr(void)
  */
 void execute_commands(void)
```

Comment:
Could you include that it should be invoked from main?

Reply:
Yeah, I thought about that, but I am not completely sure we only want to execute it from main.

Reply:
from where  then? ;)

Reply:
Maybe from anywhere (from the interruption generated on message received). This could allow us to stop the mouse while running. I don't know, maybe if it goes crazy and starts spinning non-stop like it sometimes does... :joy: